### ISSUE TITLE
Figure out non-Ubuntu Gobject dependencies for WebKit2 bindings

### ISSUE THREAD

**Issue by ncoghlan** [pybee/toga-gtk#10]
I think for Fedora the WebKit2 bindings are included by default in gobject-introspection, so folks shouldn't need to install anything extra (I didn't need to once I figured out the "3.0" vs "4.0" binding version problem and updated to the newer commit that allows either).

Debian should be the same package names as Ubuntu, just needs checking as to which versions align with which Debian releases.

RHEL & CentOS will be a bit trickier to check, as that requires using either the 3.4 stack in EPEL or Software Collections, and I'm pretty sure neither of those setups includes the Workstation pieces at the moment.

**Comment by apollo13** [comment]
Yeah debian/ubuntu have `gir1.2-webkit2-4.0` and `gir1.2-webkit2-3.0` respectively.

**Comment by apollo13** [comment]
At least CentOS7 (and therefor RHEL) should provide it as `libwebkit2gtk` (I didn't test it, just searched for the gir repository file in yum on one of why servers).

**Comment by ncoghlan** [comment]
Ah, good suggestion - looking closer on Fedora 24, it seems the relevant library is `webkitgtk4`:

```
$ sudo dnf -q whatprovides /usr/lib64/girepository-1.0/WebKit2-4.0.typelib
webkitgtk4-2.12.5-1.fc24.x86_64 : GTK+ Web content engine library
```

My guess would be that that gets installed by default, but I don't actually know for sure, so it makes sense to mention it explicitly.

I also poked around a bit at the `webkitgtk3` library, and it looks like that doesn't publish `WebKit2` bindings so installing it wouldn't help with anything:

```
$ sudo dnf repoquery --list webkitgtk3 | grep gir
/usr/lib/girepository-1.0/JavaScriptCore-3.0.typelib
/usr/lib/girepository-1.0/WebKit-3.0.typelib
/usr/lib64/girepository-1.0/JavaScriptCore-3.0.typelib
/usr/lib64/girepository-1.0/WebKit-3.0.typelib
```
